sdc/tests/test_series.py

```diff
@@ -6026,7 +6028,8 @@ def test_series_islower_str(self):
     @skip_sdc_jit("Series.str.isalnum is not supported yet")
     def test_series_isalnum_str(self):
         cfunc = self.jit(isalnum_usecase)
-        test_data = test_global_input_data_unicode_kind4 + test_global_input_data_unicode_kind1
+        test_data = [test_global_input_data_unicode_kind4, test_global_input_data_unicode,
+                     test_global_input_data_unicode_kind1]
```

Comment:
I meant to import `test_global_input_data_unicode_kind1` and `test_global_input_data_unicode_kind4` from `sdc.tests.test_utils` and not use these variables initialized in this file. `test_global_input_data_unicode_kind1` needs to be added to `sdc.tests.test_utils`.